docs/tutorial/index.rst

```diff
@@ -44,14 +44,19 @@ values for the attributes ``start`` and ``end``.
 
 .. literalinclude:: examples/firstdim.py
 
-In the translation of a call to an ONNX operator, keyword arguments (aka named arguments)
-of Python are translated into attribute parameters (of ONNX), while positional arguments
+In the translation of a call to an ONNX operator, the translator makes use of the
```

Comment:
do we want to clarify this is regarding native onnx ops and not onnx functions, because even though we don't have op schemas we still have the annotations?